# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

AdjustDogPositionActionGoal action_goal
AdjustDogPositionActionResult action_result
AdjustDogPositionActionFeedback action_feedback
